src/main/java/org/spongepowered/mod/registry/SpongeGameRegistry.java

```diff
@@ -167,6 +171,14 @@
                                                                            .put("LEFT", new SpongeRotation(270))
                                                                            .put("TOP_LEFT", new SpongeRotation(315))
                                                                            .build();
+    private static final ImmutableMap<String, GameMode> gameModeMappings =  new ImmutableMap.Builder<String, GameMode>()
```

Comment:
But, that's never going to get used here.

`setGameModes` iterates over the fields in `GameModes`, and uses the name of the field as the key. Unless `NOT_SET` is added to `GameModes`, adding it here does nothing.